.github/workflows/test.yml

```diff
@@ -138,4 +138,27 @@ jobs:
         cargo test --target ${{ matrix.target }} -p test_implement_vector &&
         cargo test --target ${{ matrix.target }} -p test_implement_winrt 
       if: matrix.version == 'nightly'
+
+    - name: Check import libs
```

Comment:
Thanks! You'll need to add this to the `tool_yml` crate to generate this yml.

Reply:
✔️